.github/workflows/schemas.yml

```diff
@@ -3,10 +3,9 @@ name: Schemas
 on:
   workflow_dispatch: # Activate this workflow manually
   pull_request:
```

Comment:
Do we still need the `pull_request` event?

Reply:
Yes, that's the trigger that makes the cURL call which in turn triggers the workflow in the other repo

Reply:
I will split the openapi and pipeline YAML generation. OpenAPI should go with the REST API in the long run, and still needs `pull_request`.